gmplot/gmplot.py

```diff
@@ -274,19 +282,45 @@ def write_map(self,  f):
             '\t\tvar map = new google.maps.Map(document.getElementById("map_canvas"), myOptions);\n')
         f.write('\n')
 
-    def write_point(self, f, lat, lon, color, title):
+    def write_point(self, f, lat, lon, color, title, label):
         f.write('\t\tvar latlng = new google.maps.LatLng(%f, %f);\n' %
                 (lat, lon))
         f.write('\t\tvar img = new google.maps.MarkerImage(\'%s\');\n' %
                 (self.coloricon % color))
         f.write('\t\tvar marker = new google.maps.Marker({\n')
+        f.write('\t\tlabel: \'%s\',\n' % label)
```

Comment:
The label sits a bit low in the marker:

![image](https://user-images.githubusercontent.com/8406475/80339539-ddd6c580-882c-11ea-88a6-ba1d8134c65d.png)

Modifying the [`labelOrigin`](https://developers.google.com/maps/documentation/javascript/reference/marker#Icon) to move the label higher should solve this.